src/chapters/html/structure.rst

```diff
@@ -0,0 +1,79 @@
+HTML Structure
+==============
+
+.. index:: ! head , ! title , ! html , ! doctype , ! body
+
+Programmers should follow certain rules about how to structure a HTML file.
+The rules about how to structure a HTML file and the tags used to lay out this structure are vital to the browser being able to render the page.
+
+Structure Rules
+---------------
+
+When it comes to laying out the overarching structure of a HTML file, a programmer should follow 5 rules:
+
+1. Every HTML file needs a ``DOCTYPE`` tag, specifying that the file is in fact, HTML. This is the only tag that does not require a closing tag.
+2. The ``<html>`` tag denotes the beginning and end of the HTML the programmer has written. 
+3. The ``<head>`` tag contains everything that goes at the head of the document.
```

Comment:
Possibly change to "The ``<head>`` tag contains information about or data about the content in <body>"

Reply:
Reworded slightly to "The ``<head>`` tag contains data about the web page."